archive_feedstock.py

```diff
@@ -54,14 +56,26 @@ def feedstocks(directory="archive"):
 
 
 def main(owner="conda-forge", check_only=False):
-    for feedstock in feedstocks("archive"):
-        print(f"Archiving {owner}/{feedstock}-feedstock...")
-        archive_repo(owner, f"{feedstock}-feedstock", archive=True, check_only=check_only)
-        time.sleep(0.5)
-    for feedstock in feedstocks("unarchive"):
-        print(f"Unarchiving {owner}/{feedstock}-feedstock...")
-        archive_repo(owner, f"{feedstock}-feedstock", archive=False, check_only=check_only)
-        time.sleep(0.5)
+    exceptions = []
+    for task in "archive", "unarchive":
+        seen = set()
+        for feedstock in feedstocks(task):
+            verb = "Archiving" if task == "archive" else "Unarchiving"
+            print(f"{verb} {owner}/{feedstock}-feedstock...", end=" ")
```

Comment:
It looks like the verb is getting cut short. For [example]( https://github.com/conda-forge/admin-requests/actions/runs/3845806223/jobs/6550384893#step:6:12 ):

```python
Error: ng conda-forge/retworkx-feedstock... [ERROR]
```

Reply:
I think that's GH's rendering detecting `[ERROR]` and messing with the output?

Reply:
Let's see if I can use something else.

Reply:
It might also be that `print`ing is being buffered and so things are getting ordered in an odd way. If we disable buffering, this may help

Reply:
Added that too just in case!

Reply:
Interesting, looks like you were right. GHA is interpreting and modifying the line somehow. Using `!!` [works]( https://github.com/conda-forge/admin-requests/actions/runs/3845967004/jobs/6550724600#step:6:10 )

```python
Archiving conda-forge/doesnotexist-feedstock... [!!]
```

Reply:
Appears [this is intentional]( https://github.blog/2020-09-23-a-better-logs-experience-with-github-actions/#opening-the-door-to-a-more-colorful-experience ). Also looks like there is a way to [affect this behavior]( https://stackoverflow.com/a/71091060 )